here-naksha-lib-handlers/src/main/java/com/here/naksha/lib/handlers/DefaultViewHandler.java

```diff
@@ -98,7 +93,7 @@ protected EventProcessingStrategy processingStrategyFor(IEvent event) {
 
       if (properties.getSpaceIds() == null || properties.getSpaceIds().isEmpty()) {
         logger.error("No spaces configured, so can't process this request");
-        return new ErrorResult(XyzError.NOT_FOUND, "No spaces configured for handler.");
+        return new ErrorResponse(NakshaError.NOT_FOUND, "No spaces configured for handler.", null, null);
```

Comment:
I think we can introduce new `ErrorResponse` constructor that would have default values (`null`) for two last parameters - I see `.., null, null` a lot